### DISCUSSION TITLE
Dshidmini Application wont open. (windows 10 and 11)

### DISCUSSION THREAD

**Discussion by socomlover** [nefarius/DsHidMini#195] [Support]
Im having trouble getting the application to open even after installing the INF files.
tried windows 10 fully updated, and also windows 11 fully updated.

I cant seem to get it working at all.  help please!

**Comment by nefarius** [comment]
Is there any DSHMC related error in the Event Viewer under the Application logs? The INF files have nothing to do whether DSHMC can launch or not, it's a .NET application that depends on Windows built-in components only.

**Reply by socomlover** [reply]
its giving me a couple things that pop up.
Trying to open app #1: Application Error.
Trying to open app #2:  Source: .NET runtime, Description:  The process was termintaed due to an unhandled exception.

and alot of other info that came with it
like faulting mode paths ect.

**Reply by nefarius** [reply]
Yeah.... All that "other information" would've been really valuable to folks like me so thanks for omitting it...
